src/OrderFormAcceptanceTests.Steps/Steps/PreviewOrderSummary.cs

```diff
@@ -424,6 +433,20 @@ public void ThenTheTotalMonthlyCostIsExpressedAsTwoDecimalPlaces()
             actual.Should().Be(expectedDecimalPointLength);
         }
 
+        [Given(@"multiple order items with different service recipient have been added to the order")]
+        public void GivenMultipleOrderItemsWithDifferentServiceRecipientHaveBeenAddedToTheOrder()
+        {
+            var order = (Order)Context["CreatedOrder"];
+            var serviceRecipient = new ServiceRecipient().Generate(order.OrderId, "yolo", "some name");
+            serviceRecipient.Create(Test.ConnectionString);
+
+            GivenACatalogueSolutionWithAFlatPriceVariableOn_DemandOrderTypeWithTheQuantityPeriodPerYearIsSavedToTheOrder();
+
+            order.OrganisationOdsCode = "yolo";
+
+            GivenACatalogueSolutionWithAFlatPriceVariableOn_DemandOrderTypeWithTheQuantityPeriodPerYearIsSavedToTheOrder();
```

Comment:
does this work, do you not need to update the context order with the new information?

Reply:
Yep this works, the context order is a reference to the order so modifying it intrinsically updates the context 😄